### ISSUE TITLE
Complete system lock-up

### ISSUE THREAD

**Issue by XxX69Krhal69XxX** [CleverRaven/Cataclysm-DDA#42588]
# Describe the bug
CDDA is causing my entire computer to lock-up. Can't access alt-tab or Ctr + alt + delete. Left it running for 30 minutes and it didn't unlock, had to do a hard reboot. This has happened 3 times now but I can't figure out the trigger. If I follow my exact steps before the lock-up it wont trigger. No crash log is created. 

# Steps To Reproduce
N/A

# Expected behavior

# Screenshots

# Versions and configuration

 - OS: Windows 10
 - Game Version: 0.E-4634-g9bdb5cc (tiles) (10882 in launcher) 
 - Ingame language: English
 - Mods loaded: disable npc needs

# Additional context

Here's my save file
[manual_backup.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/5005143/manual_backup.zip)

**Comment by andrei8l** [comment]
Can you please list those exact steps too?

**Comment by XxX69Krhal69XxX** [comment]
From the last save

1 - Turn around
2 - Squeeze past the turrets using the south west corner
3 - Pull hand brake, turn off car, get out and shoot the zombie attacking the turret
4 - pulp the zombie. Grab the deputy badge and kevlar vest
5 - Wear the vest and badge
6 - return to vehicle and proceed south down the road

Then the game locked up shortly after going down the road. I should point out because I don't think I made it very clear in my first post. The game has locked up 3 separate times in different places with nothing that seems to connect them together. First time was near a military base surrounded by a minefield, the second was in a open field and the last happened shortly after I made that back-up  (I was save scumming in case the turrets killed me) after rebooting I repeated what I did before and it didn't lock up. Maybe it has something to do with objects being spawned as they enter the reality bubble because you'll find different things along such as cars along the road as you move.
